### ISSUE TITLE
Make the autoupdater popup clearly marked with where it's from

### ISSUE THREAD

**Issue by scarletcafe** [powercord-org/powercord#622]
I'm going to be honest, this little popup is sinister as hell and when I saw it at first I thought I'd gotten some strange malware.

![image](https://user-images.githubusercontent.com/8049998/160604586-c58b1d48-4b06-45d2-b415-de9af8ca2a97.png)

I'd like to request that you mark this (and just in general anything that can pop up to obscure your screen) with some powercord marking or text, because it otherwise isn't obvious where it comes from and I really hate when software does that.

**Comment by 12944qwerty** [comment]
The fork is literally on the top left....

**Comment by scarletcafe** [comment]
I don't think this is good enough. It doesn't match the canonical powercord logo, and it's not like I sit around all day looking at the logo to remember and recognize it in the wild anyway (besides, it's also not a particularly unique logo and is easily confusable for just an emoji or other action icon anyhow).
